pages/wiki/docserver.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Local containerized documentation server
+layout: documentation
+---
+
+This web site can be built and run within a software container, such as those used *container engines* such as [Docker](https://www.docker.com/) or [podman](https://podman.io/). One reason for doing this is to be able to install and build the documentation without having to load the collection of tools on your real computer. Building and running a documentation server in a local container is the topic of this page.
+
+# Building the containerized version
+---
+
+This documentation will use `podman`, but the commands for `docker` are identical. If you wish to use `docker` instead of `podman`, just substitute `docker` for `podman` in each of the commands shown below. Any differences are explicitly noted.
+
+``` Bash
+podman build https://raw.githubusercontent.com/AsteroidOS/asteroidos.org/master/Dockerfile -t asteroidos-docserver
+```
+
+This will create an `asteroidos-docserver` image which is only around 236 MB and based on the Alpine Linux distribution. To run it locally:
+
+``` Bash
+podman run -p8080:80 asteroidos-docserver
+```
+
+This will run a server in the container which can then be reached on your local machine by pointing your browser to `localhost:8080/`.
+
+# Running with modifications
```

Comment:
This looks like the same thing repeated a third time now, maybe you can consolidate both sections ?

It's very likely that anyone bringing up asteroidos.org locally does it to introduce modifications so it's fair to consider this the default workflow

Reply:
That's a good point.  I'll combine then and reduce the duplication.